apps/ledger-live-desktop/src/renderer/hooks/useActionCards.tsx

```diff
@@ -31,6 +32,12 @@ const useActionCards = () => {
     const currentCard = findCard(cardId);
     const actionCard = findActionCard(cardId);
 
+    if (actionCard?.isMock) {
+      dispatch(
```

Comment:
same as for the portfolio cards, you can take the `setActionCards` at the line 47 out of the `if  (currentCard)` condition